common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -20,7 +20,7 @@
               label_class = 'response-label field-label label-inline'
             %>
 
-            <label id="${id}-${choice_id}-label"
+            <label id="label-${id}-${choice_id}"
```

Comment:
What is this ID being referenced by? If it's not referenced by anything, let's delete.